<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>tsucard</class>
 <widget class="QWidget" name="tsucard">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>140</width>
    <height>190</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>140</width>
    <height>190</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>140</width>
    <height>190</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>1</x>
     <y>1</y>
     <width>140</width>
     <height>190</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>140</width>
     <height>190</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>140</width>
     <height>190</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">/*background: qlineargradient( x1:0 y1:0, x2:1 y2:0, stop:0 cyan, stop:1 blue);*/
QFrame#frame {
    background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:0, y2:1, stop:0 rgba(90, 90, 90, 255), stop:0.1 rgba(51, 51, 51, 255), stop:1 rgba(51, 51, 51, 255));}

QFrame#frame {
    border-width: 1px;
    border-style: solid;
    border-radius: 15px;
}
</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <property name="lineWidth">
    <number>2</number>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_3">
    <property name="spacing">
     <number>1</number>
    </property>
    <property name="leftMargin">
     <number>1</number>
    </property>
    <property name="topMargin">
     <number>1</number>
    </property>
    <property name="rightMargin">
     <number>1</number>
    </property>
    <property name="bottomMargin">
     <number>1</number>
    </property>
    <item>
     <layout class="QHBoxLayout" name="head">
      <property name="spacing">
       <number>0</number>
      </property>
      <item>
       <widget class="QLabel" name="lblTitle">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="minimumSize">
         <size>
          <width>110</width>
          <height>18</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>110</width>
          <height>18</height>
         </size>
        </property>
        <property name="font">
         <font>
          <pointsize>8</pointsize>
          <kerning>true</kerning>
         </font>
        </property>
        <property name="cursor">
         <cursorShape>PointingHandCursor</cursorShape>
        </property>
        <property name="styleSheet">
         <string notr="true">color: rgb(255, 255, 255);
margin: 1px, 0px, 0px, 6px;
background-color:transparent;</string>
        </property>
        <property name="lineWidth">
         <number>0</number>
        </property>
        <property name="text">
         <string>ShortName</string>
        </property>
        <property name="textFormat">
         <enum>Qt::PlainText</enum>
        </property>
        <property name="alignment">
         <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
        </property>
        <property name="indent">
         <number>-1</number>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnCancel" native="true">
        <property name="minimumSize">
         <size>
          <width>18</width>
          <height>18</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>18</width>
          <height>18</height>
         </size>
        </property>
        <property name="cursor">
         <cursorShape>PointingHandCursor</cursorShape>
        </property>
        <property name="styleSheet">
         <string notr="true">border:none;
padding:0px;
background-repeat: no-repeat;
margin: 0px, 0px, 2px, 0px;
background-color:transparent;
background-image: url(:/images/x.png);
background-position: center center;</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="Line" name="line">
      <property name="styleSheet">
       <string notr="true">background-color: rgb(82, 82, 82);
margin: 2px,4px,2px,0px;</string>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="info">
      <property name="spacing">
       <number>0</number>
      </property>
      <item>
       <widget class="QWidget" name="imgStreaming" native="true">
        <property name="minimumSize">
         <size>
          <width>40</width>
          <height>40</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>40</width>
          <height>40</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">border:none;
background-repeat: no-repeat;
background-position: center center;
background-image: url(:/images/tsu_streaming_off.png);
background-color:transparent;</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout">
        <property name="spacing">
         <number>0</number>
        </property>
        <property name="sizeConstraint">
         <enum>QLayout::SetDefaultConstraint</enum>
        </property>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QLabel" name="lblTotal">
            <property name="minimumSize">
             <size>
              <width>56</width>
              <height>10</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>56</width>
              <height>10</height>
             </size>
            </property>
            <property name="font">
             <font>
              <pointsize>6</pointsize>
              <weight>75</weight>
              <bold>true</bold>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">color: rgb(130, 130, 130);
background-color:transparent;
padding-top:3px;
padding-right:5px</string>
            </property>
            <property name="lineWidth">
             <number>0</number>
            </property>
            <property name="text">
             <string>Total</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
            </property>
            <property name="margin">
             <number>0</number>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="txtTotal">
            <property name="minimumSize">
             <size>
              <width>22</width>
              <height>10</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>22</width>
              <height>10</height>
             </size>
            </property>
            <property name="font">
             <font>
              <family>MS Shell Dlg 2</family>
              <pointsize>8</pointsize>
              <weight>75</weight>
              <bold>true</bold>
              <stylestrategy>PreferAntialias</stylestrategy>
              <kerning>true</kerning>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">color: rgb(3, 215, 19);
background-color:transparent;</string>
            </property>
            <property name="lineWidth">
             <number>0</number>
            </property>
            <property name="text">
             <string>729</string>
            </property>
            <property name="scaledContents">
             <bool>false</bool>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="txtTotalUnit">
            <property name="minimumSize">
             <size>
              <width>17</width>
              <height>10</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>17</width>
              <height>10</height>
             </size>
            </property>
            <property name="font">
             <font>
              <pointsize>6</pointsize>
              <weight>75</weight>
              <bold>true</bold>
              <kerning>true</kerning>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">color: rgb(3, 215, 19);
background-color:transparent;
padding-top:3px</string>
            </property>
            <property name="lineWidth">
             <number>0</number>
            </property>
            <property name="text">
             <string>MB</string>
            </property>
            <property name="margin">
             <number>0</number>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QLabel" name="lblDownloaded">
            <property name="minimumSize">
             <size>
              <width>56</width>
              <height>10</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>56</width>
              <height>10</height>
             </size>
            </property>
            <property name="font">
             <font>
              <pointsize>6</pointsize>
              <weight>75</weight>
              <bold>true</bold>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">color: rgb(130, 130, 130);
background-color:transparent;padding-top:3px;padding-right:5px</string>
            </property>
            <property name="lineWidth">
             <number>0</number>
            </property>
            <property name="text">
             <string>Downloaded</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="txtDownloaded">
            <property name="minimumSize">
             <size>
              <width>22</width>
              <height>10</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>22</width>
              <height>10</height>
             </size>
            </property>
            <property name="font">
             <font>
              <pointsize>8</pointsize>
              <weight>75</weight>
              <bold>true</bold>
              <strikeout>false</strikeout>
              <stylestrategy>PreferAntialias</stylestrategy>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">color: rgb(253, 194, 1);
background-color:transparent;</string>
            </property>
            <property name="lineWidth">
             <number>0</number>
            </property>
            <property name="text">
             <string>521</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="txtDownloadedUnit">
            <property name="minimumSize">
             <size>
              <width>17</width>
              <height>10</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>17</width>
              <height>10</height>
             </size>
            </property>
            <property name="font">
             <font>
              <pointsize>6</pointsize>
              <weight>75</weight>
              <bold>true</bold>
              <strikeout>false</strikeout>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">color: rgb(253, 194, 1);
background-color:transparent;
padding-top:3px</string>
            </property>
            <property name="lineWidth">
             <number>0</number>
            </property>
            <property name="text">
             <string>MB</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <widget class="QLabel" name="lblRemaining">
            <property name="minimumSize">
             <size>
              <width>56</width>
              <height>11</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>56</width>
              <height>11</height>
             </size>
            </property>
            <property name="font">
             <font>
              <pointsize>6</pointsize>
              <weight>75</weight>
              <bold>true</bold>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">color: rgb(130, 130, 130);
background-color:transparent;padding-top:2px;padding-right:5px;</string>
            </property>
            <property name="lineWidth">
             <number>0</number>
            </property>
            <property name="text">
             <string>Remaining</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
            </property>
            <property name="margin">
             <number>0</number>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="txtRemaining">
            <property name="minimumSize">
             <size>
              <width>22</width>
              <height>10</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>22</width>
              <height>10</height>
             </size>
            </property>
            <property name="font">
             <font>
              <pointsize>8</pointsize>
              <weight>75</weight>
              <bold>true</bold>
              <stylestrategy>PreferAntialias</stylestrategy>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">color: rgb(11, 129, 225);
background-color:transparent;</string>
            </property>
            <property name="lineWidth">
             <number>0</number>
            </property>
            <property name="text">
             <string>208</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="txtRemainingUnit">
            <property name="minimumSize">
             <size>
              <width>17</width>
              <height>10</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>17</width>
              <height>10</height>
             </size>
            </property>
            <property name="font">
             <font>
              <pointsize>6</pointsize>
              <weight>75</weight>
              <bold>true</bold>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">color: rgb(11, 129, 225);
background-color:transparent;
padding-top:3px</string>
            </property>
            <property name="lineWidth">
             <number>0</number>
            </property>
            <property name="text">
             <string>MB</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QGridLayout" name="circle">
      <property name="spacing">
       <number>0</number>
      </property>
      <item row="0" column="0">
       <widget class="QWidget" name="bkgContainet" native="true">
        <property name="minimumSize">
         <size>
          <width>132</width>
          <height>60</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>132</width>
          <height>60</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">border:none;
padding:0px;
background-repeat: no-repeat;
background-color:transparent;</string>
        </property>
        <widget class="QWidget" name="bkgArrow" native="true">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>12</y>
           <width>132</width>
           <height>37</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">border:none;
padding:0px;
background-repeat: no-repeat;
background-position: center center;
background-color:transparent;
background-image: url(:/images/bkg1.png);</string>
         </property>
        </widget>
        <widget class="QWidget" name="bkgCircle" native="true">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>0</y>
           <width>132</width>
           <height>60</height>
          </rect>
         </property>
         <property name="minimumSize">
          <size>
           <width>132</width>
           <height>60</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>132</width>
           <height>60</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">border:none;
padding:0px;
background-repeat: no-repeat;
background-position: center center;
background-image: url(:/images/pb3.png);
background-color:transparent;</string>
         </property>
        </widget>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="footer">
      <property name="spacing">
       <number>0</number>
      </property>
      <item>
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QWidget" name="widget" native="true">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>30</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>30</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">border:none;
padding:0px;
background-repeat: no-repeat;
background-color:transparent;
background-image: url(:/images/bkg2.png);
</string>
        </property>
        <widget class="QPushButton" name="btnPause">
         <property name="geometry">
          <rect>
           <x>27</x>
           <y>7</y>
           <width>15</width>
           <height>15</height>
          </rect>
         </property>
         <property name="cursor">
          <cursorShape>PointingHandCursor</cursorShape>
         </property>
         <property name="styleSheet">
          <string notr="true">border:none;
background-image: url(:/images/pause.png);
padding:0px;
background-repeat: no-repeat;
background-position: center center;
background-color:transparent;
</string>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="iconSize">
          <size>
           <width>15</width>
           <height>15</height>
          </size>
         </property>
        </widget>
        <widget class="QPushButton" name="btnResume">
         <property name="geometry">
          <rect>
           <x>44</x>
           <y>7</y>
           <width>15</width>
           <height>15</height>
          </rect>
         </property>
         <property name="cursor">
          <cursorShape>PointingHandCursor</cursorShape>
         </property>
         <property name="styleSheet">
          <string notr="true">border:none;
background-image: url(:/images/resume.png);
padding:0px;
background-repeat: no-repeat;
background-position: center center;
background-color:transparent;
</string>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="iconSize">
          <size>
           <width>15</width>
           <height>15</height>
          </size>
         </property>
        </widget>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
